release/models/qos/openconfig-qos-interfaces.yang

```diff
@@ -321,6 +321,12 @@ submodule openconfig-qos-interfaces {
       description
         "Number of packets dropped by the queue due to overrun";
     }
+ 
+    leaf dropped-octets {
```

Comment:
I think we initially excluded the octet counter here as it wasn't clear that implementations count the size of packets that they dropped, that decision was a long time ago though, are there implementations with this today? (I'm happy to include it.)

Reply:
Yes, I'd like to include it.

Reply:
This sounds good to me. Can you link to these MIBs in the description of the PR please? (To demonstrate support for them across implementations.)

Reply:
@robshakir  done, please review.